data/templatetags/ribbon_link.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+from community.git import get_remote_url
```

Comment:
This should be in community/ module

Reply:
Okay, will do in some days, I have some exams going on.

Reply:
Do you mean the folder where git.py resides or a new folder named `module` (due to templatetag)?

Reply:
Move this file to `community/templatetags/ribbon_link.py`

Reply:
Done.